.github/workflows/code-health.yml

```diff
@@ -37,6 +37,8 @@ jobs:
   unit-test:
     needs: build
     runs-on: ubuntu-latest
+    permissions:
```

Comment:
sorry I deleted this in a previous PR. do you mind to add a comment referencing the info

Reply:
sure, updated